database/database.go

```diff
@@ -0,0 +1,183 @@
+package database
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"v9_deployment_manager/worker"
+
+	"github.com/google/uuid"
+)
+
+type Driver struct {
+	db *sql.DB
+}
+
+func CreateDriver(psqlInfo string) (*Driver, error) {
+	db, err := sql.Open("postgres", psqlInfo)
+	if err != nil {
+		return nil, err
+	}
+	return &Driver{
+		db: db,
+	}, nil
+}
+
+// TODO: Consider using transactions throughout here
+
+func (driver *Driver) FindUserID(githubUsername string) (string, error) {
+	var userID string
+	upsertQuery := `INSERT INTO v9.public.users(email, github_username) VALUES (NULL, $1)
```

Comment:
Add documentation for why you're replacing the github username with...the same username